### PR TITLE
openrussian-cli: init at 1.0.0

### PR DESCRIPTION
**Pull request by vs49688**
###### Motivation for this change

Adding the `openrussian-cli` package.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -11308,4 +11308,14 @@
     github = "zupo";
     githubId = 311580;
   };
+  zane = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:11311]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

File: pkgs/misc/openrussian-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    wrapProgram $out/bin/openrussian \
+      --prefix LUA_PATH ';' "$LUA_PATH" \
+      --prefix LUA_CPATH ';' "$LUA_CPATH"
+
+    runHook postInstall
+  '';
+
+  postInstall = ''
+    installShellCompletion --bash --name openrussian ./openrussian-completion.bash
```
Comment (SuperSandro2000) [inline pkgs/misc/openrussian-cli/default.nix:51]:
```suggestion
    installShellCompletion --cmd openrussian --bash ./openrussian-completion.bash
```

File: pkgs/misc/openrussian-cli/default.nix [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+  meta = with lib; {
+    homepage    = "https://github.com/rhaberkorn/openrussian-cli";
+    description = "Offline Console Russian Dictionary (based on openrussian.org)";
+    license     = with licenses; [ gpl3 mit cc-by-sa-40 ];
+    maintainers = with maintainers; [ zane ];
+    platforms   = platforms.unix;
+  };
+}
+
```
Comment (SuperSandro2000) [inline pkgs/misc/openrussian-cli/default.nix:63]: One final new line to much.

Comment (vs49688) [reply pkgs/misc/openrussian-cli/default.nix]: Fixed

File: pkgs/misc/openrussian-cli/default.nix (hunk 2 above)
Comment (vs49688) [reply pkgs/misc/openrussian-cli/default.nix]: Fixed

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (vs49688) [reply maintainers/maintainer-list.nix]: Fixed, but the list isn't entirely sorted. It's above `zseri`, but the next one is `pulsation`. I hope that's alright.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120012` at ca3681e1 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>openrussian-cli</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall`.

Near pkgs/misc/openrussian-cli/default.nix:38:3:
```
   |
38 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

makeWrapper is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/misc/openrussian-cli/default.nix:22:3:
```
   |
22 |   buildInputs = [ makeWrapper ] ++ luaLibs;
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/misc/openrussian-cli/default.nix:58:5:
```
   |
58 |     license     = with licenses; [ gpl3 mit cc-by-sa-40 ];
   |     ^
```
</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix]: There are some entries at the end that need sorting.
